libs/utils/utils.js

```diff
@@ -1253,7 +1242,6 @@ async function processSection(section, config, isDoc) {
   preloadBlockResources(section.preloadLinks);
   await Promise.all([
     decoratePlaceholders(section.el, config),
-    decorateIcons(section.el, config),
```

Comment:
Here was a great place for this to live in in order to load everything in parallel, for which we've seen good real world benefits on the CRuX LCP data 
![image](https://github.com/user-attachments/assets/ed41ba73-2a59-4040-abdb-d1bb152188b1)